CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Portal field name|DATS schema field|Status|Description|Portal help text|
+|-|-|-|-|-|
+|Title|title|**REQUIRED**| The name of the dataset. |The name of the dataset. The title needs to be short and easily recognizable and searchable. If an acronym, please also provide the long name.|
+|Creators|creators|**REQUIRED**| The person(s) or organization(s) which contributed to the creation of the dataset. | The person(s) or organization(s) which contributed to the creation of the dataset. This can be the principal investigator, hospital, university, centre, clinic, etc. The first contact provided will be associated as the main contact for this dataset.|
+|Description|description|**REQUIRED**| A textual narrative comprised of one or more statements describing the dataset.| A short paragraph providing a rapid overview of the dataset and the context of data collection. Suggestion of items to include in the description (if applicable): main use of the dataset, population studied, study design, sample size, data collected, methods, techniques, apparatus used to generate the data.|
+|Data types|types|**REQUIRED**| Terms to describe the nature of the data. | Terms to describe the nature of the data. Data type can be single or multiple. Add a term with the [interlex URI](https://neuinfo.org/interlex/dashboard) if possible.| 
+|Version|version|**REQUIRED**| Provide the version number, or the release point of your dataset.|Provide the version number, or the release point of your dataset. Need to be a positive number. |
+|Licenses|licenses|**REQUIRED**| The use of license name abbreviations is suggested for specifying a license. Please visit [Creative Commons](https://creativecommons.org/share-your-work/) to choose the right licence for you.| The licences under which this dataset is shared.|
+|Keywords|keywords|**REQUIRED**| Tags associated with the dataset, which will help in its discovery. We suggest entering at least 5 keywords, different from the datatype.|Tags associated with the dataset, which will help in its discovery. These should be well known terms by the research community. |
+|Formats|distributions - format|**REQUIRED**| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite)Primary data files format (*example*: csv, nifti, txt, fasta).| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite).|
+|Size|distributions - size|**REQUIRED**| Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
+|Units|distributions - unit|**REQUIRED**| Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(MB -> MegaByte, GB -> GiGaByte, TB -> TeraByte and PB -> PetaByte). |
+|Access |||The information about access modality for the dataset distribution.|The information about access modality for the dataset distribution.|
```

Comment:
I am not sure I understand that sentence. Maybe modify for

```suggestion
|Access |||The information about access modality for the dataset distribution.|Access information for the dataset distribution.|
```

Reply:
@emmetaobrien thoughts on this?

Reply:
At the moment we have precise descriptions for every field within `access` so I would suggest that we do not actually need this line.

Reply:
This is the help comment just under the 'Access' title., hence there is no DATS schema field.
I think it should be kept here or completely removed from the GUI

Reply:
I would be for removing it from the GUI.

Reply:
removed!